terraform/gitlab/gitlab.tf.json.template.py

```diff
@@ -1497,6 +1500,42 @@ def qq(*words):
                 'depends_on': ['aws_iam_service_linked_role.gitlab_ssm']
             }
         },
+        'aws_cloudformation_stack': {
+            # Images whose short name begins with '_' are only used outside the
+            # security boundary, so vulnerabilities that are detected within
+            # them do not need to be addressed with the same urgency.
+            #
+            # Using CF stack because the AWS provider does not support
+            # Inspector filters and the AWSCC provider fails due to
+            # https://github.com/hashicorp/terraform-provider-awscc/issues/1364
+            'inspector_filters': {
```

Comment:
Technically, you would need to create a dependency on the enabler resource, but due to above comment you can't do that now.  But please capture that aspect in the FIXME issue.

Reply:
https://github.com/DataBiosphere/azul/issues/5760